docs/source/how-to-downstream.mdx

```diff
@@ -56,14 +50,14 @@ Begin by retrieving your file URL with [`hf_hub_url`], and then pass the specifi
 '/home/lysandre/.cache/huggingface/hub/bc0e8cc2f8271b322304e8bb84b3b7580701d53a335ab2d75da19c249e2eeebb.066dae6fdb1e2b8cce60c35cc0f78ed1451d9b341c78de19f3ad469d10a8cbb1'
 ```
 
-[`hf_hub_url`] and [`cached_download`] work hand in hand to download a file. This is precisely how [`hf_hub_download`] from the tutorial works! [`hf_hub_download`] is simply a wrapper that calls both [`hf_hub_url`] and [`cached_download`].
+[`hf_hub_url`] and [`cached_download`] work hand-in-hand to download a file. This is precisely how `hf_hub_download` from the tutorial works! `hf_hub_download` is simply a wrapper that calls both `hf_hub_url` and `cached_download`.
```

Comment:
Shouldn't the following `hf_hub_download`, `hf_hub_url` etc also contains the brackets?

Reply:
Oops, thanks for catching that! I was originally going to remove the mention of `hf_hub_download` since we don't really have a tutorial. But I think I will just keep that here, introduce it in the quick start guide, and then link to it.